src/Build/Resources/Strings.resx

```diff
@@ -1063,6 +1063,10 @@
       LOCALIZATION: "{2}" is a localized message from a CLR/FX exception. Also, Microsoft.Build.Framework should not be localized
     </comment>
   </data>
+  <data name="TaskReturnedFalseButDidNotLogError">
+    <value>MSB4132: Build failed because the "{0}" task returned false but did not log an error.</value>
```

Comment:
```suggestion
    <value>MSB4132: The "{0}" task returned false but did not log an error.</value>
```

(task failures don't _always_ fail the build in the presence of `ContinueOnError`.